src/sxastarter/src/components/LayoutContainer.tsx

```diff
@@ -45,12 +47,23 @@ export const Default = (props: ComponentProps): JSX.Element => {
 
   const id = props.params.RenderingIdentifier;
 
+  let columnStackClass = '';
+  const stackAtBreakpoint = props.params.StackColumnAt;
```

Comment:
Rendering parameters might be empty or underfined sometimes, let's also cover those

Reply:
Updated!